### DISCUSSION TITLE
Why does download_to_dir return ‘Download_Limit_Reached.txt’ even with minimal downloads?

### DISCUSSION THREAD

**Discussion by yyq83** [metaspace2020/metaspace#1604] [Q&A]
Why does the function [download_to_dir](https://metaspace2020.readthedocs.io/en/latest/content/apireference/sm_annotation_utils.html?highlight=download_to_dir#metaspace.sm_annotation_utils.SMDataset.download_to_dir) always download a file named “Download_Limit_Reached.txt” when I use it, even though I’ve only attempted to download one file?

**Comment by Bisho2122** [comment]
Recently, we noticed attempts to download all datasets in bulk from METASPACE, which prompted us to take this protective measure and restrict downloads for each user. This restriction limits downloads to ensure fair use and maintain the integrity of the datasets.

If you are interested in downloading more than one dataset, we ask you to submit a request outlining the datasets you're interested in downloading to contact@metaspace2020.eu . The development team will review the request and access will be granted accordingly.

Thank you for your understanding and apologies for the inconvenience. Please don't hesitate to contact us if you have further questions or need assistance with the request process.
